src/ovp_pipeline/commands/backfill_twitter_authors.py

```
@@ -0,0 +1,234 @@
+
+
+def _resolve_api_key(cli_arg: str | None, vault_dir: Path) -> str | None:
+    if cli_arg:
+        return cli_arg.strip()
+    env = os.environ.get("TWITTERAPI_IO_KEY")
+    if env:
+        return env.strip()
+    keyfile = vault_dir / "60-Logs" / ".twitterapi-io-key"
+    if keyfile.exists():
+        for line in keyfile.read_text(encoding="utf-8").splitlines():
+            line = line.strip()
```

Ruff (0.15.12)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)